internal/component/database_observability/mysql/collector/query_sample.go

```diff
@@ -264,24 +261,26 @@ func (c *QuerySample) fetchQuerySamples(ctx context.Context) error {
 			continue
 		}
 
-		cpuTime := float64(row.CPUTime) / 1e9
-		elapsedTime := float64(row.ElapsedTimePicoseconds.Int64) / 1e9
-
-		logMessage := fmt.Sprintf(
-			`schema="%s" digest="%s" digest_text="%s" rows_examined="%d" rows_sent="%d" rows_affected="%d" errors="%d" max_controlled_memory="%db" max_total_memory="%db" cpu_time="%fms" elapsed_time="%fms" elapsed_time_ms="%fms"`,
-			row.Schema.String,
-			row.Digest.String,
-			digestText,
-			row.RowsExamined,
-			row.RowsSent,
-			row.RowsAffected,
-			row.Errors,
-			row.MaxControlledMemory,
-			row.MaxTotalMemory,
-			cpuTime,
-			elapsedTime,
-			elapsedTime,
-		)
+		cpuTime := picosecondsToMilliseconds(row.CPUTime)
+		elapsedTime := picosecondsToMilliseconds(row.ElapsedTimePicoseconds.Float64)
+
+		logMessage :=
+			fmt.Sprintf(
+				`schema="%s" digest="%s" digest_text="%s" rows_examined="%d" rows_sent="%d" rows_affected="%d" errors="%d" max_controlled_memory="%db" max_total_memory="%db" cpu_time="%fms" elapsed_time="%fms" elapsed_time_ms="%fms" time="%dms" level="INFO"`,
+				row.Schema.String,
+				row.Digest.String,
+				digestText,
+				row.RowsExamined,
+				row.RowsSent,
+				row.RowsAffected,
+				row.Errors,
+				row.MaxControlledMemory,
+				row.MaxTotalMemory,
+				cpuTime,
+				elapsedTime,
+				elapsedTime,
+				row.TimestampMilliseconds,
```

Comment:
I think we should do this as a followup, given it will need refactoring to `buildLokiEntry`. But ideally we set the timestamp of the log entry 
https://github.com/grafana/alloy/blob/main/internal/component/database_observability/mysql/collector/loki_entry.go#L21 to `row.TimestampMilliseconds`